src/hl/group.rs

```diff
@@ -167,6 +169,36 @@ impl Group {
         let name = to_cstring(name)?;
         Dataset::from_id(h5try!(H5Dopen2(self.id(), name.as_ptr(), H5P_DEFAULT)))
     }
+
+    /// Non-recursively gets all the members in the group
```

Comment:
Maybe "Returns names of all the members in the group, non-recursively."? (also a missing period)

Reply:
Done.